### PR TITLE
Clarify where layers are pulled from.

### PR DESCRIPTION
**Pull request by AdamIsrael**
This merge addresses the question raised in issue #703

### CODE REVIEW

**Comment by marcoceppi** [comment]
LGTM with one nit

File: src/en/authors-charm-building.md [hunk 1]
```diff
@@ -225,7 +225,8 @@ charm.  From within the layer directory, this is as simple as:
 charm build
 ```
 
-Build will take all of the layers and create a new charm into
+Build will take all of the layers, looking first in your local LAYER_PATH and
+then querying interfaces.juju.solutions, and create a new charm into
```
Comment (marcoceppi) [inline src/en/authors-charm-building.md]: could we hyperlink the interfaces page?

**Comment by evilnick** [comment]
@AdamIsrael if you fix up the above we can merge this before it goes stale!

**Comment by pmatulis** [comment]
@AdamIsrael 
Assuming your changes are still valid, can you add the missing bits so we can merge this?

**Comment by chuckbutler** [comment]
@pmatulis @evilnick if you merge this as is i'll follow up with a hyperlink since this is quite old and decaying further.

**Comment by evilnick** [comment]
\o/

**Pull request merged**